supportingami/sami-website
# Ft resources schema

#### What does this PR do?
- Features changes to the resource database type names, as well as abstraction of the resource query.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout _ft-resources-schema_ or update local branch. Run the server and populate a resource to see it on the frontend.

#### What are the relevant tickets?
- None
